notmuchfs.c

```diff
@@ -1147,39 +1149,34 @@ static int notmuchfs_unlink (const char* path)
 
  if (last_pslash != NULL) {
    char *last_slash = strrchr(path, '/');
-   char  trans_name[PATH_MAX];
 
    strncpy(trans_name, last_slash + 1, PATH_MAX - 1);
    trans_name[PATH_MAX - 1] = '\0';
    string_replace(trans_name, '#', '/');
 
+   path = trans_name;
+ }
+
 #if 0
-   /* Delete the message from the notmuch database too. Is this the right
```

Comment:
Why? This indentation is now inconsistent with the 2 spaces used everywhere else. Please undo this hunk.